loader.js

```diff
@@ -1261,20 +1261,20 @@ var Module = null;
        //  chunk of prebuffered audio. At that point it seems like
        //  JSMESS never catches up and our sound glitches forever.
 
-       var insert_point = (buffer_insert_point === null)
-         ? now
-         : buffer_insert_point;
+       var insert_point = (buffer_insert_point === null) ?
+         now :
+         buffer_insert_point;
```

Comment:
maybe I could add a .jshintrc file in order to disable this and other unnecessary warnings.